frontend/src/breeding_form.tsx

```diff
@@ -419,10 +441,29 @@ export function BreedingForm({
               )}
               // jscpd:ignore-end
 
-              onChange={(event: any, newValue: OptionType | null) => {
-                newValue && setFormField("father", newValue.value);
+              onChange={(event: any, newValue: LimitedIndividual | null) => {
+                newValue && setFormField("father", newValue.number);
               }}
             />
+            {is_admin ||
+              (is_owner && (
+                <KeyboardDatePicker
+                  autoOk
+                  variant="inline"
+                  inputVariant={inputVariant}
+                  disableFuture={false}
+                  className="simpleField"
+                  label="Äldsta födelsedatum på far"
```

Comment:
I think this label might be confusing. What about "Begränsa listan på hanar till individer födda innan" ? (if this isn't too long...)
Also, layout wise, I'd try to set this above the father-dropdown.

Reply:
Another option is to just hide it 10 years old alive rabbit should be enough